kibana.md

```diff
@@ -399,6 +399,28 @@ The result may look something like this:
 http://localhost:5601/login?nextUrl=%2Fapp%2Fkibana%23%2Fvisualize%2Fedit%2F28dcde30-2258-11e8-82a3-af58d04b3c02%3F_g%3D%28%29&jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWV9.TJVA95OrM7E2cBab30RMHrHDcEfxjoYZgeFONFh7HgQ
 ```
 
+## Audit log
+Audit log feature is wide described in [📖docs for Elasticsearch plugin](elasticsearch.md#audit-logs). 
```

Comment:
> **The** audit log feature is **widely** described in ...

Remember to use the articles (please add them also to the rest of the PR) :)

Reply:
Did you mean to use `# Audit log`, '## Loading visualization', and so on?